cmd/manager/chartrepos.go

```diff
@@ -14,18 +14,38 @@ func registerChartRepoRoutes(c *router.Context, h *router.Handler) {
 
 func listChartRepositoriesHandlerFunc(w http.ResponseWriter, r *http.Request, c *router.Context) error {
 	handler := "manager: list chart repositories"
-	util.LogHandlerExitWithJSON(handler, w, "a repo here", http.StatusOK)
+	repos, err := c.Manager.ListChartRepos()
+	if err != nil {
+		return err
+	}
+	util.LogHandlerExitWithJSON(handler, w, repos, http.StatusOK)
```

Comment:
At this point, we should be using `httputil.Encode()` instead of this. Feel free to do that in a separate PR, though, since that change needs to be made in a number of places.